### PR TITLE
Update the schema of meta.yaml

### PR DESCRIPTION
**Pull request by monaka**
### What does this PR do?

Applies more strict rules to the schema file for meta.yaml

### CODE REVIEW

**Comment by monaka** [comment]
Currently, builds will be failed. But it is expected behavior. they are blocked by eclipse/che-plugin-registry#242

**Comment by monaka** [comment]
@amisevsk Thank you for the heads up. I'll resolve conflicts and repush within a few days.

File: v3/plugins/eclipse/che-theia/7.3.0/meta.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ spec:
          - name: HOSTED_PLUGIN_HOSTNAME
            value: 0.0.0.0
          - name: HOSTED_PLUGIN_PORT
-           value: 3130
+           value: "3130"
```
Comment (nickboldt) [inline v3/plugins/eclipse/che-theia/7.3.0/meta.yaml]: is it worth updating 7.3.0 when 7.3.1, 7.3.2 and 7.4.0 are live?

Comment (monaka) [reply v3/plugins/eclipse/che-theia/7.3.0/meta.yaml]: Checker rejects without patching this ...

```
Checking plugin 'eclipse/che-theia/7.3.0'
        3130 is not of type u'string'
```

Comment (monaka) [reply v3/plugins/eclipse/che-theia/7.3.0/meta.yaml]: This is similar issue with https://github.com/eclipse/che-plugin-registry/pull/242 .

Comment (nickboldt) [reply v3/plugins/eclipse/che-theia/7.3.0/meta.yaml]: ah, then +1.

**Comment by monaka** [comment]
@amisevsk I forgot to apply #249 related changes. I fixed and re-pushed.

**Pull request merged**